site/gatsby-site/static/discover/src/algolia.js

```diff
@@ -1,32 +1,30 @@
+/* eslint-disable */
```

Comment:
Consider the old discover app as deprecated. Let's only make changes to it if something is broken.